docker-compose.yml

```diff
@@ -31,7 +55,7 @@ services:
             - /srv/app/vendor
         environment:
             - APP_ENV=dev
-            - APP_LIBERO_API=http://api-dummy-nginx
+            - APP_LIBERO_API=http://article-store:8000
```

Comment:
if you merge `master` into this branch, this configuration will end up in https://github.com/libero/walking-skeleton/blob/master/docker-compose.yml#L32 as there's a gateway now